tsdb/docs/format/chunks.md

```diff
@@ -36,7 +36,7 @@ in-file offset (lower 4 bytes) and segment sequence number (upper 4 bytes).
 
 Notes:
 * `<uvarint>` has 1 to 10 bytes.
-* `encoding`: Currently either `XOR` or `histogram`.
+* `encoding`: Currently either `XOR`, `histogram`, or `float histogram`.
```

Comment:
`float histogram` → `floathistogram` (to match the string returns of `Encoding.String()` in chunkenc/chunk.go).

Reply:
I forgot I made this comment when merging. I'll include the change in https://github.com/prometheus/prometheus/pull/15135 .